### PR TITLE
docs update

### PR DESCRIPTION
**Pull request by Ellenn-A**
# Updates
- added Implicit invitation under verification flow

### CODE REVIEW

File: README_VERITABLE.md [hunk 1]
```diff
@@ -448,7 +448,89 @@ Using this credential ID `POST http://localhost:3001/credentials/{credentialReco
...
+  "x": "A35kxICNVG-ICgG4PMx4W6QZcA7Yo07nvBajotrBqIs"
+}
+```
+
+In order to send an invite from bob to Alice, Alice needs to be aware of the private key from keypair above and our did (above).
+We need to import our did with the private key on Alice using endpoint `dids/import`:
+
+```json
+{
+  "did": "did:web:Ellenn-A.github.io:dids:1",
```
Comment (jonmattgray) [inline README_VERITABLE.md:512]: replace with YOUR_USERNAME?

File: README_VERITABLE.md [hunk 2]
```diff
@@ -448,7 +448,89 @@ Using this credential ID `POST http://localhost:3001/credentials/{credentialReco
...
+      "verificationMethod": [
+        {
+          "id": "did:web:YOUR_USERNAME.github.io:dids:1#owner",
+          "type": "JsonWebKey2020",
+          "controller": "did:web:YOUR_USERNAME.github.io:dids:1",
+          "publicKeyJwk": {
+            "kty": "OKP",
+            "crv": "Ed25519",
+            "x": "A35kxICNVG-ICgG4PMx4W6QZcA7Yo07nvBajotrBqIs"
+          }
```
Comment (jonmattgray) [inline README_VERITABLE.md:471]: how was this part generated?

**Review by msterle — COMMENTED** [review]
Good work, but private keys should never be stored on git, please replace this web:did with a new key immediately. Despite this being a temporary did only used for testing, this exposes risk and is bad practice.

Can you please instead describe the process for publishing a web:did, and remove this from git history?

**Comment by mattdean-digicatapult** [comment]
> Good work, but private keys should never be stored on git, please replace this web:did with a new key immediately. Despite this being a temporary did only used for testing, this exposes risk and is bad practice.

Completely agree. Pleas emake sure this key is not used or referenced anywhere else as it's now been compromised. Feel free to leave the did document in here as an example fo what it looks like but blank out the private key with something like `x`s or similar.

> Can you please instead describe the process for publishing a web:did, and remove this from git history?

Note removing this from the history will not remove the key from the git object files, so in this instance probably doesn't do anything overly helpful. If we really want to remove this we'd need to contact github if I remember correctly @dblane-digicatapult

**Review by jonmattgray — APPROVED** [review]

**Pull request merged**